cypress/integration/userCanLogin.spec.js

```diff
@@ -0,0 +1,68 @@
+describe('User can log in to application', () => {
+  it('successfully', () => {
+    cy.server()
+    cy.route({
+      method: 'POST',
+      url: 'http://localhost:3000/auth/sign_in',
+      response: 'fixture:successful_user_login.json',
+      status: 200
+    })
+    cy.route({
+      method: 'GET',
+      url: 'http://localhost:3000/v1/trails',
+      response: 'fixture:user_can_view_list_of_trails.json'
+    })
```

Comment:
You can put this and cy.server() into `cypress/support/index.js` file and then you don't have to call it in every test file/suite.

This is an example:

https://github.com/CraftAcademy/gnosis-client/blob/development/cypress/support/index.js

Reply:
Done! Really cool thanks!